app/containers/Admin/OrganisationRequestsTable.tsx

```diff
@@ -0,0 +1,218 @@
+        <Row>
+          <Col md={2}>
+            <Dropdown style={{width: '100%'}}>
+              <Dropdown.Toggle
+                style={{width: '100%', backgroundColor: '#036'}}
+                id="dropdown-sort"
+              >
+                {orderByDisplay}
+              </Dropdown.Toggle>
+              <Dropdown.Menu style={{width: '100%'}}>
+                {dropdownSortItems.map(item => (
+                  <DropdownMenuItemComponent
+                    key={item.eventKey}
+                    itemId="sortApplications"
+                    itemEventKey={item.eventKey}
+                    itemFunc={(eventKey, event) => handleEvent(event, eventKey)}
+                    itemTitle={item.title}
+                  />
+                ))}
+                ;
+              </Dropdown.Menu>
+            </Dropdown>
+          </Col>
+          <Col md={1}>
+            <Button
+              id="toggleDirection"
+              style={{width: '100%', backgroundColor: '#036'}}
+              onClick={event => handleEvent(event)}
+            >
+              {direction}
+            </Button>
+          </Col>
+          <Col md={1} />
+          <Col md={2}>
+            <Dropdown style={{width: '100%'}}>
+              <Dropdown.Toggle
+                style={{width: '100%', backgroundColor: '#036'}}
+                id="dropdown-filter"
+              >
+                {searchDisplay}
+              </Dropdown.Toggle>
+              <Dropdown.Menu style={{width: '100%'}}>
+                {[
+                  {eventKey: null, title: 'No Filter'},
+                  ...dropdownSortItems
+                ].map(item => (
+                  <DropdownMenuItemComponent
+                    key={item.eventKey}
+                    itemId="applySearchField"
+                    itemEventKey={item.eventKey}
+                    itemTitle={item.title}
+                    itemFunc={(eventKey, event) => handleEvent(event, eventKey)}
+                  />
+                ))}
+                ;
+              </Dropdown.Menu>
+            </Dropdown>
+          </Col>
+          <Col md={6}>
+            <Form id="applySearchValue" onSubmit={event => handleEvent(event)}>
+              <Form.Row>
+                <Form.Group as={Col} md={8}>
+                  <Form.Control type="string" />
+                </Form.Group>
+                <Form.Group as={Col} md={1}>
+                  <Button type="submit" style={{backgroundColor: '#036'}}>
+                    Filter
+                  </Button>
+                </Form.Group>
+              </Form.Row>
+            </Form>
+          </Col>
+        </Row>
+      </Container>
+      <br />
+      <br />
+      <Table
+        striped
+        hover
+        style={{textAlign: 'center', border: '1px solid #f5f5f5'}}
+      >
+        <thead style={{backgroundColor: '#036', color: 'white'}}>
+          <tr>
+            <th>#</th>
+            <th>Name</th>
+            <th>Lastname</th>
```

Comment:
We should have the user email in this table. Name and last name are not enough to identify a user